packages/task/src/browser/task-definition-registry.spec.ts

```diff
@@ -15,6 +15,7 @@
  ********************************************************************************/
 
 import { expect } from 'chai';
+import { foobarTaskFixture } from '../node/test/test-helper';
```

Comment:
@magician-margatroid `browser` (frontend) should not import anything from `node` (backend), even if it is simply for test purposes: [code organization](https://github.com/eclipse-theia/theia/wiki/Code-Organization). Given that the `foobarTaskFixture` is simply for test purposes, you can include the contents directly in the `task-definition-registry.spec.ts` file.

Reply:
Thank you very much @vince-fugnitto . Excuse me and I'll improve it.